### ISSUE TITLE
DJango 2.05 migration fails with on_delete=SET_NULL but field cannot be null

### ISSUE THREAD

**Issue by davmlaw** [arneb/django-messages#112]
DJango==2.0.5

```
django_messages.Message.sender: (fields.E320) Field specifies on_delete=SET_NULL, but cannot be null.
	HINT: Set null=True argument on the field, or change the on_delete rule.
```

Solution  Either make the field nullable, or on_delete=DO_NOTHING - not sure what you want to do

**Comment by PhilBrooks** [comment]
Given the reasoning for this change in #105 was to prevent a deleted user from cascading down and deleting the message itself, and given the recipient is nullable, I'd say the better solution is to make the sender field nullable.
Would you agree, @arneb ?

**Comment by frankystone** [comment]
The same problem appears with Django 1.11.12 if you want to setup a new database (sqlite3).

**Comment by WebVictory** [comment]
Django == 2.0.6
The application does not work.
TypeError: __init__() missing 1 required positional argument: 'on_delete'
Traceback (most recent call last):
  File "C:\Program Files (x86)\JetBrains\PyCharm 2016.3.1\helpers\pycharm\django_manage.py", line 41, in <module>
    run_module(manage_file, None, '__main__', True)
  File "C:\Python36\lib\runpy.py", line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "C:\Python36\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\Python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:/skript/untitled7\manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "C:\Python36\lib\site-packages\django\core\management\__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "C:\Python36\lib\site-packages\django\core\management\__init__.py", line 347, in execute
    django.setup()
  File "C:\Python36\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Python36\lib\site-packages\django\apps\registry.py", line 112, in populate
    app_config.import_models()
  File "C:\Python36\lib\site-packages\django\apps\config.py", line 198, in import_models
    self.models_module = import_module(models_module_name)
  File "C:\Python36\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "C:\Python36\lib\site-packages\django_messages\models.py", line 48, in <module>
    class Message(models.Model):
  File "C:\Python36\lib\site-packages\django_messages\models.py", line 54, in Message
    sender = models.ForeignKey(AUTH_USER_MODEL, related_name='sent_messages', verbose_name=_("Sender"))

**Comment by arneb** [comment]
fixed in 0.6.0
